### DISCUSSION TITLE
Guidance on creating a theme

### DISCUSSION THREAD

**Discussion by AndrewCraydenCER** [wet-boew/wet-boew#9608] [Q&A]
Hello, the Canada Energy Regulator is interested in extending WET through a custom theme. Is there a recommend way to create a WET theme, while continuously keeping it up to date with the latest version of WET? Any documentation surrounding this would be much appreciated 🙏

**Comment by ghost** [comment]
@duboisp would probably be best person to connect with.

But currently there only two themes as others where abandoned as no one took up maintaining them. You can learn more about the old themes at https://github.com/wet-boew/wet-boew/issues/8366.

If you want the theme to be officially supported it would basically have to be under the wet-boew umbrella and someone would need to maintain the issues, pull request, and releases.

Otherwise you have to look at building a custom one which would involve css and/or js. They’re no starting guide or tool to assist as far as I’m aware.

When I’m near my computer I could probably link some sites that have wet-boew as it main js but custom theme.

###### I am just a volunteer on this GitHub project.
